addons/account/models/account_move_line.py

```diff
@@ -1690,22 +1690,22 @@ def _compute_display_name(self):
             )
 
     def copy_data(self, default=None):
-        data_list = super().copy_data(default=default)
+        vals_list = super().copy_data(default=default)
```

Comment:
Is this diff really needed? :thinking:

Reply:
This is for consistency purpose (as developers tend to copy/paste existing code).